tests/integration/test_telemetry.py

```diff
@@ -0,0 +1,52 @@
+import platform
+
+import pytest
+
+from msgraph.core import SDK_VERSION, APIVersion, GraphClient, NationalClouds
+
+BASE_URL = NationalClouds.Global + '/' + APIVersion.v1
+
+
+@pytest.fixture
+def graph_client():
```

Comment:
Please add a test that demonstrates that the customer can set client-request-id on the the request and get it in the response.

Reply:
Done